### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by xp44mm**
estabilished -> established
mimetype -> mime type // there has been 'mime types'
stictly -> strictly

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ internal-links.tap
 stats.json
 printable.md
 repositories/*.json
+/.vs
```
Comment (snitin315) [inline .gitignore:16]: Please revert this change.

**Review by snitin315 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (xp44mm) [reply .gitignore:16]: I don't know how to do it.

Comment (snitin315) [reply .gitignore:16]:
```suggestion
```

**Review by chenxsan — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]
